jupyter_scheduler/handlers.py

```diff
@@ -206,6 +207,33 @@ async def delete(self, job_id):
         self.finish()
 
 
+class JobFromDefinitionHandler(ExtensionHandlerMixin, JobHandlersMixin, APIHandler):
+    @tornado.web.authenticated
+    async def post(self, job_definition_id: str):
+        payload = self.get_json_body()
+        try:
+            model = CreateJobFromDefinition(**payload)
+            job_id = await ensure_async(
+                self.scheduler.create_job_from_definition(job_definition_id, model=model)
+            )
+        except InputUriError as e:
```

Comment:
It doesn't hurt this to be there, but haven't we removed the input URI altgoether?

Reply:
Correct, this should not apply here.